src/Libraries/Microsoft.Extensions.Caching.Hybrid/Microsoft.Extensions.Caching.Hybrid.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Multi-level caching implementation building on and extending IDistributedCache</Description>
+    <!--
+      relevant TFM feature summary (all "nice to have", with reasonable workarounds):
+        nca 2.0 / ns 2.1: RuntimeHelpers.IsReferenceOrContainsReferences; Task.IsCompletedSuccessfully; HashCode
+        nca 3.0 : IThreadPoolWorkItem
+        net 5.0 : [DynamicallyAccessedMembers], EncodingExtensions.GetString/GetBytes, IsExternalInit
+        net 7.0 : ArgumentNullException.ThrowIfNull
+    -->
+    <TargetFrameworks>$(NetCoreTargetFrameworks)$(ConditionalNet462);net8.0;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
I don't believe explicit `net8.0` is required. `$(NetCoreTargetFrameworks)` should be `net9.0;net8.0`, isn't this the case?

Reply:
Ah, I must have looked at [`main`](https://github.com/dotnet/extensions/blob/main/Directory.Build.props#L16) rather than [`dev`](https://github.com/dotnet/extensions/blob/dev/Directory.Build.props#L17C66-L17C73); yes, you're right, in `dev`, net8.0 is included - will remove (for context: the intent here is "also include LTS, because Aspire targets LTS" - if that happens to match the intent of "extensions", then: great!)

Reply:
Yep, we're targeting the LTS and the most recent supported STS.